src/plugins/data/common/search/aggs/utils/get_aggs_formats.ts

```diff
@@ -143,9 +143,11 @@ export function getAggsFormats(getFieldFormat: GetFieldFormat): FieldFormatInsta
           return params.otherBucketLabel;
         }
 
+        const joinTemplate = ` ${params.separator ?? '-'} `;
```

Comment:
We changed default from ` > ` to `-` is this expected?

Reply:
I think this change happened before this commit who change the default from `-` to `>`: https://github.com/elastic/kibana/pull/116928/commits/9ef5e19573a41dfbd279ab8b1c0f10afe0f9531d

I can set `>` as default. Do you think it is still worth leaving the option?

Reply:
I'd leave the option and keep ` > ` as default (note whitespaces around it)